packages/core/src/controllers/OptionsController.ts

```diff
@@ -5,6 +5,7 @@ import type { CustomWallet, Metadata, ProjectId, SdkVersion, Tokens } from '../u
 // -- Types --------------------------------------------- //
 export interface OptionsControllerState {
   projectId: ProjectId
+  allWallets?: 'show' | 'hide' | 'onlyMobile'
```

Comment:
To stick with naming of other feature flags, could we rename this to `enableAllWallets`.  Perhaps constants can be uppercase as well i.e `SHOW`, `HIDE`, `ONLY_MOBILE` etc..

Reply:
Wouldn't it be a bit confusing?
The enable flag is currently only in the default wagmi config, while this is createWeb3Modal, also all the enable flags are booleans while this has string options?